pkg/build/api/v1beta1/conversion.go

```diff
@@ -125,6 +127,7 @@ func init() {
 		// Rename DockerBuildStrategy.BaseImage to DockerBuildStrategy.Image
 		func(in *newer.DockerBuildStrategy, out *DockerBuildStrategy, s conversion.Scope) error {
 			out.NoCache = in.NoCache
+			out.PullSecretName = in.PullSecretName
```

Comment:
need this on the other dockerbuilderstrategy converter too.  (the old to new one)